core/dropdowndiv.js

```diff
@@ -192,7 +197,23 @@ Blockly.DropDownDiv.getContentDiv = function() {
  */
 Blockly.DropDownDiv.clearContent = function() {
   Blockly.DropDownDiv.content_.innerHTML = '';
-  Blockly.DropDownDiv.content_.style.width = '';
+  Blockly.DropDownDiv.content_.style.cssText = '';
```

Comment:
Can we check on the compatibility of cssText in IE11? https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/cssText